lib/DxilContainer/DxilContainerAssembler.cpp

```diff
@@ -941,10 +941,11 @@ class DxilPSVWriter : public DxilPartWriter {
       break;
     }
     case ShaderModel::Kind::Compute: {
-      UINT waveSize = (UINT)m_Module.GetWaveSize();
-      if (waveSize != 0) {
-        pInfo->MinimumExpectedWaveLaneCount = waveSize;
-        pInfo->MaximumExpectedWaveLaneCount = waveSize;
+      UINT waveMinSize = (UINT)m_Module.GetMinWaveSize();
+      UINT waveMaxSize = (UINT)m_Module.GetMaxWaveSize();
```

Comment:
Ok, now we have another ordering of the words, Wave, Size, and Min/Max/Preferred.  We can fix this later, since it's internal, but I'd prefer we use WaveSize(Min/Max/Preferred), and even better, roll these into a struct that you can access directly.  Then struct can have methods, like HasRange(), which abstracts the degenerate encoding away from each bit of code that has to deal with this.